app/src/marketplace/modules/dataUnion/services.js

```diff
@@ -164,19 +164,19 @@ export const getJoinPartStreamId = (address: DataUnionId, usePublicNode: boolean
     call(getCommunityContract(address, usePublicNode).methods.joinPartStream())
 
 export const getDataUnionStats = (id: DataUnionId): ApiResult<Object> => get({
-    url: formatApiUrl('communities', id, 'stats'),
+    url: formatApiUrl('dataunions', id, 'stats'),
     useAuthorization: false,
 })
 
 export const getDataUnions = async (): ApiResult<Array<Object>> => {
-    const { communities } = await get({
-        url: formatApiUrl('communities'),
+    const { dataunions } = await get({
```

Comment:
Hmm, I wonder if data union server has renamed `communities` to `dataunions`? Might be that we need to revert this change as I couldn't find any changes related to renaming on data union server side.

Reply:
@tumppi 

- [This change](https://github.com/streamr-dev/engine-and-editor/pull/736) adds a `/dataunions` route to `engine-and-editor` and deprecates `/communities`
- [This change](https://github.com/streamr-dev/streamr-community-products/pull/39) adds a `/dataunions` route alias to the data unions server and deprecates `/communities`

Reply:
I see @hpihkala! Still seems that the actual response contains property `communities` instead of `dataunions` (https://github.com/streamr-dev/streamr-community-products/blob/300d874ec31c457b745c7799ef0a9061bfa97612/src/routers/communities.js#L31).

Until that is fixed, we need to revert this destructuring assignment to old property name `communities`.

Reply:
@tumppi ah I see what you mean! Yeah, it's unfortunate that the response content uses the old terminology. There's [this PR](https://github.com/streamr-dev/streamr-community-products/pull/40) but it's a big one that addresses both external and internal renaming. It might take a while to get merged. I've left there a recommendation to split it into two PRs: one for external and one for internal changes.

As to work around the immediate issue here, a few questions:
- What is the list of data unions actually used for in the UI?
- For backwards compatibility, could you pick either `communities` or `dataunions` from the returned object, depending on which one is present? With a deprecation notice to remove the old alternative once it's no longer present.

Reply:
That endpoint is used in marketplace product listing to show member count badge on the corner of each data union product. And yeah, we could pick up both properties from the response and see which one is defined 👍 I'll make a change.